### PR TITLE
aruha-580: Provide online access to OpenAPI reference definition

### PR DESCRIPTION
**Pull request by adyach**
This PR brings ability to view api swagger file using https://github.com/zalando-stups/twintip-spring-web

### CODE REVIEW

**Comment by lmontrieux** [comment]
:shipit:

**Comment by rcillo** [comment]
@adyach is there a way to keep a symbolic link from the old file to the new file. We have dozens of users that refer to the API directly through the link and this link will be broken due to renaming.

**Comment by rcillo** [comment]
@adyach if there is no way to keep this link, we better search for everywhere that links to this file, like our techjira, readme, nakadi manual, nakadi libraries...

**Comment by adyach** [comment]
@rcillo good point, i will check this.

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -78,6 +78,12 @@ nakadi:
   auth:
     plugin:
       factory: org.zalando.nakadi.plugin.auth.DefaultApplicationServiceFactory
+
+twintip:
+  mapping: /api
+  ui: /api/ui/
```
Comment (antban) [inline src/main/resources/application.yml]: What does it mean? I mean what is /api/ui and /api ?

Comment (adyach) [inline src/main/resources/application.yml]: http://stups.readthedocs.io/en/latest/components/twintip.html

**Comment by rcillo** [comment]
:shipit:

**Comment by rcillo** [comment]
:shipit:

**Comment by antban** [comment]
:+1:

**Pull request merged**